compiler/aarch64/codegen/GenerateInstructions.cpp

```diff
@@ -157,3 +157,57 @@ TR::Instruction *generateMemSrc1Instruction(TR::CodeGenerator *cg, TR::InstOpCod
       return new (cg->trHeapMemory()) TR::ARM64MemSrc1Instruction(op, node, mr, sreg, preced, cg);
    return new (cg->trHeapMemory()) TR::ARM64MemSrc1Instruction(op, node, mr, sreg, cg);
    }
+
+TR::Instruction *generateArithmeticShiftRightImmInstruction(TR::CodeGenerator *cg, TR::Node *node,
+   TR::Register *treg, TR::Register *sreg, uint32_t shiftAmount, bool is64bit, TR::Instruction *preced)
```

Comment:
Can you not determine whether this is 64-bit from the IL opcode on the node?

Reply:
I didn't think of using the data size from the Node.  I am going to change it.